egui/src/widgets/plot.rs

```diff
@@ -332,6 +365,12 @@ impl Plot {
         self
     }
 
+    /// If true, the bounds will be set based on the data.
+    pub fn automatic_bounds(mut self, enabled: bool) -> Self {
```

Comment:
This is a bit under-documented (what is the default? what does it mean for it to be `false`?).

I wonder if we really need this at all? I like the behavior of automatically calculating bounds at the start and on double-click, but otherwise allow users to scroll and zoom.